source/triggers/database-triggers.txt

```diff
@@ -463,118 +553,29 @@ importing an application directory with {+cli-bin+}.
 
 .. _last-cluster-time-processed:
 
-Last Cluster Time Processed
----------------------------
-
-All Triggers keep track of the last time the Trigger executed, known as
-the **latest execution** in the App Services UI. Database Triggers also keep track
-of the **last cluster time processed**: the last time the change stream
-backing that Trigger emitted an event. You can access these timestamps
-in the list of Triggers that displays when you first navigate to the
-:guilabel:`Triggers` section from the sidebar.
-
-The difference between the latest execution and the last cluster time
-processed represents the latency between when an event occurs in your
-database, and when the Trigger finishes responding to that event. If
-latency is consistently large, you can reduce it with the following
-mitigations:
```

Comment:
I'm okay removing this since we dont display latency but the steps to address that are still valid, ie this bit:

> If latency is consistently large, you can reduce it with the following mitigations:
> 
> - Optimize the Trigger's linked Function to speed up execution.
> 
> - If your Trigger uses Event Ordering, consider switching ordering off
>   to reduce latency.
> 
> - Add resources by upgrading your MongoDB Atlas cluster.

Reply:
Agreed, but unless we discuss how to know if there is latency, I think it's weird to say "to fix latency...". Any thoughts on that?

Reply:
I think all of these are already covered in more detail in the "Performance Optimization" section.